registry/registry_test.go

```diff
@@ -142,7 +142,7 @@ func (*viewSuite) TestNewRegistry(c *C) {
 			c.Assert(err.Error(), Equals, tc.err, cmt)
 		} else {
 			c.Assert(err, IsNil, cmt)
-			c.Check(registry, Not(IsNil), cmt)
+			c.Check(registry, NotNil, cmt)
```

Comment:
for next time: it's best to do this kind of small tweaks/fixes as their own PR or commit with something like "drive-by" in the commit message

Reply:
Thanks, I'll keep this in mind for future changes.